packages/plugin-e2e/CONTRIBUTING.md

```diff
@@ -54,6 +54,19 @@ npm run build # used to build @grafana/plugin-e2e
 npm run dev # watches for changes to files and rebuilds @grafana/plugin-e2e automatically
 ```
 
+#### Running e2e tests locally
+
+Some of the e2e tests involves data source plugins that integrates with third-party services. To be able to run these tests, you need to add a `/packages/plugin-e2e/.env` file and provide the necessary credentials. See `/packages/plugin-e2e/.env.example` to get an understanding of what variables you need to provide.
```

Comment:
For Grafanistas it might be useful to mention we have a plugin provisioning repo?

Also I guess this could be a bit of a hurdle for any external contributor to the project?

Reply:
Yes good point. Took another stab at this. Now, all e2e tests that requires credentials are tagged as `integration`. Running the script `playwright:test` now only executes the ones that don't require credentials. There's a new script - `playwright:test:integration` - that only runs those that integrates with third-party services. Also updated the contributing guidelines. Hopefully this makes things clearer for community and Grafanista contributors.